src/coreclr/nativeaot/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -1151,6 +1151,8 @@ REDHAWK_PALEXPORT bool PalGetMaximumStackBounds(_Out_ void** ppStackLowOut, _Out
     {
 #if defined(HOST_WASM) && !defined(FEATURE_WASM_MANAGED_THREADS)
         PalGetMaximumStackBounds_SingleThreadedWasm(&pStackLowOut, &pStackHighOut);
+#elif defined(HOST_WASM) && defined(FEATURE_WASM_MANAGED_THREADS)
+        PalGetMaximumStackBounds_MultiThreadedWasm(&pStackLowOut, &pStackHighOut);
```

Comment:
This should use the `pthread_attr_init` code from below instead of a bespoke version.

Reply:
Thanks, have updated this and removed `PalGetMaximumStackBounds_MultiThreadedWasm` with the logic move to `pthread_attr_init` and `pthread_attr_getstack`